io.gitlab.librewolf-community.json

```diff
@@ -0,0 +1,75 @@
+{
+    "app-id": "io.gitlab.librewolf-community",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "40",
+    "sdk": "org.gnome.Sdk",
+    "command": "librewolf",
+    "modules": [
+        "shared-modules/dbus-glib/dbus-glib-0.110.json",
+        {
+            "name": "librewolf",
+            "buildsystem": "simple",
+            "build-commands": [
+                "mkdir -p /app/lib",
+                "mv librewolf.tar.bz2 /app/lib/librewolf",
+                "install -D -m644 /app/lib/librewolf/browser/chrome/icons/default/default128.png /app/share/icons/hicolor/128x128/apps/io.gitlab.librewolf-community.png",
+                "install -D -m644 /app/lib/librewolf/browser/chrome/icons/default/default128.png /app/usr/share/icons/hicolor/128x128/apps/io.gitlab.librewolf-community.png",
```

Comment:
why this install? `/app/usr/share` isn't  cared by Flatpak

Reply:
Without it, flatpak-builder complained about not being able to find the icon file – after some frustration and a bit of googling I came upon some issues like https://github.com/hughsie/appstream-glib/issues/271 and https://github.com/flatpak/flatpak-builder/issues/228 and found that to be at least a "somewhat reasonable" solution :)